src/openvtb/components/taken/api/tests/test_betaaltaak.py

```diff
@@ -163,6 +174,65 @@ def test_valid_create(self):
                 "einddatumHandelingsTermijn": None,
                 "datumHerinnering": betaaltaak.datum_herinnering,
                 "toelichting": betaaltaak.toelichting,
+                "partijIsToegewezenAan": betaaltaak.partij_is_toegewezen_aan,
+                "medewerkerWordtBehandeldDoor": betaaltaak.medewerker_wordt_behandeld_door,
+                "zaakHoortBij": betaaltaak.zaak_hoort_bij,
+                "productHeeftBetrekkingOp": betaaltaak.product_heeft_betrekking_op,
+                "taakSoort": betaaltaak.taak_soort,
+                "details": {
+                    "bedrag": betaaltaak.details["bedrag"],
+                    "valuta": betaaltaak.details["valuta"],
+                    "transactieomschrijving": betaaltaak.details[
+                        "transactieomschrijving"
+                    ],
+                    "doelrekening": {
+                        "naam": betaaltaak.details["doelrekening"]["naam"],
+                        "iban": betaaltaak.details["doelrekening"]["iban"],
+                    },
+                },
+            },
+        )
+
+    def test_valid_create_with_external_relations(self):
+        self.assertEqual(ExterneTaak.objects.all().count(), 0)
+        data = {
+            "titel": "titel",
+            "handelingsPerspectief": "handelingsPerspectief1",
+            "partijIsToegewezenAan": "urn:maykin:partij:brp:nnp:bsn:1234567892",
+            "medewerkerWordtBehandeldDoor": "urn:maykin:medewerker:brp:nnp:bsn:1234567892",
+            "zaakHoortBij": "urn:maykin:ztc:zaak:d42613cd-ee22-4455-808c-c19c7b8442a1",
+            "productHeeftBetrekkingOp": "urn:maykin:product:cec996f4-2efa-4307-a035-32c2c9032e89",
+            "details": {
+                "bedrag": "11",
+                "transactieomschrijving": "test",
+                "doelrekening": {
+                    "naam": "test",
+                    "iban": "NL18BANK23481326",
+                },
+            },
+        }
+        response = self.client.post(self.list_url, data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(ExterneTaak.objects.all().count(), 1)
+
+        betaaltaak = ExterneTaak.objects.get()
+        self.assertEqual(
+            response.json(),
+            {
+                "url": f"http://testserver{reverse('taken:externetaak-detail', kwargs={'uuid': str(betaaltaak.uuid)})}",
+                "urn": f"urn:maykin:taken:externetaak:{str(betaaltaak.uuid)}",
```

Comment:
Still unsure how other applications will figure out where to find this object, but I guess this is still an ongoing discussion with Joeri?